src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -62,36 +53,31 @@
 import java.util.Objects;
 import java.util.Optional;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
 import java.util.function.Function;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import java.util.stream.Collectors;
 
 public class GeneralUtils {
-
+    //Constants
     private static final DecimalFormat percentageFormat = new DecimalFormat("#.##");
-    private static final Pattern userDiscrim = Pattern.compile(".+#[0-9]{4}");
     private static final Pattern timeRegex = Pattern.compile("^([0-9]*):?([0-9]*)?:?([0-9]*)?$");
 
-    private static DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("MMMM yyyy HH:mm:ss");
-
-    private static final SimpleDateFormat preciseFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SS");
-
-    private static final PeriodFormatter prettyTime = new PeriodFormatterBuilder()
-            .appendDays().appendSuffix(" Day ", " Days ")
-            .appendHours().appendSuffix(" Hour ", " Hours ")
-            .appendMinutes().appendSuffix(" Minute ", " Minutes ")
-            .appendSeconds().appendSuffix(" Second", " Seconds")
-            .toFormatter();
     private static final PeriodFormatter periodParser = new PeriodFormatterBuilder()
             .appendDays().appendSuffix("d")
             .appendHours().appendSuffix("h")
             .appendMinutes().appendSuffix("m")
             .appendSeconds().appendSuffix("s")
             .toFormatter();
-    private static final int LEVENSHTEIN_DISTANCE = 8;
 
+    //Getters
```

Comment:
Block comment this

Reply:
Whops that was a place holder for me organizing stuff